docs/msbuild-sdk.md

```diff
@@ -0,0 +1,64 @@
+Cesium MSBuild Project SDK
+--------------------------
+
+Cesium provides it's own project SDK that could be used to simplify building of Cesium programs and libraries.
+
+Cesium MSBuild SDK inherits default behavior from a `Microsoft.NET.Sdk` SDK and tries to integrate with it the same way as C# does.
+
+Cesium MSBuild SDK only supports SDK-style projects.
+
+> Note: Some of the common MSBuild properties and items those are not stated in this document could be also used in Cesium project files. Not all of them are tested so something may not work as expected.
+
+### Source files
+Source files are defined with `<Compile>` items, very similar to other .NET languages:
+```xml
+<ItemGroup>
+    <Compile Include="hello.c" />
+    <Compile Include="example.c" />
+</ItemGroup>
+```
+> Note: In the current SDK implementation, compile units should be defined explicitly, in opposite to C# `<Compile>` items.
```

Comment:
Sorry for a confusion, the text really isn't clear about what I meant. What is important here is that instead of having all `**/*.c` included by default implicitly, you need to specify them explicitly, both full names and wildcards are supported.  Will fix this note.